.github/workflows/run_unit_tests.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [macos-latest, ubuntu-latest, windows-latest]
+        os: [macos-latest, windows-latest, ubuntu-latest]
```

Comment:
Revert unnecessary change
```suggestion
        os: [macos-latest, ubuntu-latest, windows-latest]
```

Reply:
Reverted back.

Reply:
Sorry, I didn't realize the diff I was looking at was already reverting this.

Reply:
I reverted this again.